api/teams.py

```diff
@@ -33,33 +33,53 @@
 # API route that returns all teams from DATABASE
 @teams.route('/teams', methods=['GET'])
 def api_teams_get():
-    return "Fix me!"
+    return jsonify(DATABASE), 200
 
 # API route that returns a single teams from DATABASE according to the ID in the URL
 # For example /api/teams/1 will give you Ash's Team
 @teams.route('/teams/<int:id>', methods=['GET'])
 def api_teams_id_get(id):
-    return "Fix me!"
+    for teams in DATABASE:
+        if teams['id'] == id:
+            return jsonify(teams), 200
 
 # API route that creates a new team using the request body JSON and inserts it at the end of DATABASE
 @teams.route('/teams', methods=['POST'])
 def api_teams_id_post():
-    return "Fix me!"
+    global current_id
+    new_team = json.loads(request.data)
+    new_team['id'] = current_id
+    DATABASE.append(new_team)
+    current_id += 1
+    return new_team, 200
```

Comment:
We should try and add some validation here to make sure we're adding the right data.